src/me/coley/recaf/ui/component/list/RenderFormatter.java

```diff
@@ -89,10 +89,15 @@ default String getTypeStr(Type type, ConfUI options) {
 		if (!s.contains("(") && (s.length() == 1 || s.startsWith("L") || s.startsWith("["))) {
 			s = type.getClassName();
 		}
+    if(s == null)
+    {
+      System.out.println("Type " + type + " has no descriptor or class");
```

Comment:
For future pulls using the logging impl will write to the log file and console at once. This could be `WARN` level *(Recaf.INSTANCE.logging.<level-for-method-name>(message)*. 
Original source also uses tabs instead of spaces, so for consistency keeping the scheme the same across all classes would be appreciated.